wordle-web/src/views/DailyWordListView.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <v-col>
+    <v-card
+      class="align-center text-center mx-auto"
+      max-height="100%"
+      max-width="90%"
+      min-height="90%"
+      min-width="25%"
+      rounded
+    >
+      <v-card-title class="titleText">Word Of The Day Games! (Dummy Data)</v-card-title>
+      <v-card-text>
+        <v-row justify="center">
```

Comment:
I really like how these are all buttons that take you to the word for that day. Especially with the color. I think I may just rearrange this to be from top to bottom when listing all of them to make it clear the order of when the word was the word of the day.

Reply:
Yeah, the data should be fetched ordered from the axios web api call. Left to right, top to bottom. I'll make sure this is the case once the web database is populated and we can fetch stuff.